static/app/utils/onDemandMetrics/index.tsx

```diff
@@ -110,25 +131,72 @@ export const extrapolatedAreaStyle = {
   opacity: 1.0,
 };
 
-export function hasOnDemandMetricAlertFeature(organization: Organization) {
-  return organization.features.includes('on-demand-metrics-extraction');
-}
-
-export function hasOnDemandMetricWidgetFeature(organization: Organization) {
+export function OnDemandWarningIcon({
+  msg,
+  color = 'gray300',
+}: {
+  msg: React.ReactNode;
+  color?: Color;
+}) {
   return (
-    organization.features.includes('on-demand-metrics-extraction') &&
-    organization.features.includes('on-demand-metrics-extraction-experimental')
+    <Tooltip title={msg}>
```

Comment:
```suggestion
    <Tooltip skipWrapper title={msg}>
```

you'll want to skip the wrapping span here to help center the svg via flex